features/app-launch.yml

```diff
@@ -0,0 +1,10 @@
+name: Launch handler
+description: The Launch Handler API allows developers to control how a progressive web app (PWA) is launched, for example if it uses an existing window or creates a new one, and how the app's target launch URL is handled.
+spec: https://wicg.github.io/web-app-launch/
+group: progressive-web-app
+compat_features:
```

Comment:
I think we need to add `html.manifest.launch_handler` and `html.manifest.launch_handler.client_mode`.